### DISCUSSION TITLE
如何忽略某个已被定义的class

### DISCUSSION THREAD

**Discussion by LearningUnity** [unocss/unocss#3396] [Q&A] [answered]
当 container 已经被同事定义过, uno 如何过滤导出container呢
![Snipaste_2023-11-30_10-55-16](https://github.com/unocss/unocss/assets/17720234/daf48ad5-1f90-4935-9aa6-3aeca26e519f)
此处配置不能作用于class
![Snipaste_2023-11-30_10-58-51](https://github.com/unocss/unocss/assets/17720234/27f3bcac-3a1d-4aaa-8182-8e7d6638036c)

**Comment by zyyv** [comment, marked as answer]
https://unocss.dev/guide/extracting#blocklist

**Reply by LearningUnity** [reply]
thank you so mush!!!!
